compile.sh

```diff
@@ -653,15 +683,17 @@ if [ $BUILD_WEBSITE = "ON" ]; then
 else
   echo "# - No website-related executables                        #"
 fi
-echo "#                                                         #"
-echo "# You should now be able to run Widelands via             #"
-echo "# typing ./widelands + ENTER in your terminal             #"
-echo "#                                                         #"
-echo "# You can update Widelands via running ./update.sh        #"
-echo "# in the same directory that you ran this script in.      #"
+if [ -z "$RUN" ]; then
+  echo "#                                                         #"
+  echo "# You should now be able to run Widelands via             #"
+  echo "# typing ./widelands + ENTER in your terminal             #"
+  echo "#                                                         #"
+  echo "# You can update Widelands via running ./update.sh        #"
+  echo "# in the same directory that you ran this script in.      #"
+fi
 echo "###########################################################"
 
-else
+elif [ -z "$RUN" ]; then
```

Comment:
the original else was related to line 648. It was the only output if quiet was selected. we should keep this but make the same disitnction as in lines 652 to 656

Reply:
Are you sure it's necessary? `--dry-run` already printed the commands of what would have been done, so printing anything here would be a violation of `--quiet` IMO. Only without `--dry-run` makes it sense to report completion for `--quiet`.

Reply:
Well without the quiet option the output was "Congratulations. Widelands has been built successfully." (line 653) followed by all the verbose messages. without it was only "Widelands has been built succesfully" (line 697) without any further messages. Although using dry run with quiet makes not much sense, we should keep the original logic to show one single success line if quiet is selected.